std/file.d

```diff
@@ -2983,6 +2983,31 @@ auto dirEntries(string path, SpanMode mode, bool followSymlink = true)
     return DirIterator(path, mode, followSymlink);
 }
 
+/// Duplicate functionality of D1's $(D std.file.listdir()):
+unittest
+{
+  string[] listdir(string pathname)
```

Comment:
I've found when generating the Kindle version of the documentation, 4 spaces makes things run off the right edge too soon. So for documentation, I tend to keep it tighter.